src/thunderbots/software/geom/util.h

```diff
@@ -352,6 +352,19 @@ Point reflect(const Point &v, const Point &n);
 std::pair<std::optional<Point>, std::optional<Point>> raySegmentIntersection(
     Ray &ray, Segment &segment);
 
+
+/**
+ * Calculates the intersection of two Rays
+ *
+ * @param ray1: First Ray
+ * @param ray2: Second Ray
+ * @return Returns std::nullopt if no intersections exist, or if there are infinite
+ * intersections (overlapping) Returns Point if a single intersection exists. Returns
```

Comment:
Looks like the last sentence in this comment is extra? It starts talking about Segments.

Reply:
Oops, leftover of copy/paste. Removed